### PR TITLE
Add translation codes and strings for ignore plugin

### PR DESCRIPTION
**Pull request by blutarch**
Complements vanilla/addons#794.

Addresses vanilla/support#1616.

A number of strings in the ignore plugin had not been added to tx-source files.

### CODE REVIEW

File: tx-source/dash_core.php [hunk 1]
```diff
@@ -8,6 +8,7 @@
 $Definition['%s item in queue'] = '%s item in queue';
 $Definition['%s level'] = '%s level';
 $Definition['%s levels'] = '%s levels';
+$Definition['%s prohibited from using the ignore feature.'] = '%s prohibited from using the ignore feature.';
```
Comment (tburry) [inline tx-source/dash_core.php:11]: This doesn't seem grammatically correct.

File: tx-source/dash_core.php [hunk 2]
```diff
@@ -406,6 +407,7 @@
 $Definition['HTML Emails'] = 'HTML Emails';
 $Definition['HTTP Strict Transport Security (HSTS) Settings'] = 'HTTP Strict Transport Security (HSTS) Settings';
 
+$Definition['IgnoreListRevoke'] = 'Revoke <b>%s</b>\'s ignore list privileges?';
```
Comment (tburry) [inline tx-source/dash_core.php:410]: It seems like all of these strings are in the frontend and not the dashboard. If that is the case then these go in site_core.

File: tx-source/dash_core.php (hunk 1 above)
Comment (blutarch) [reply tx-source/dash_core.php]: It creates a grammatically correct sentence when populated by the $ReferTo variable. Is this something you want changed?

File: tx-source/site_core.php [hunk 3]
```diff
@@ -51,6 +51,7 @@
 $Definition['%s hour'] = '%s hour';
 $Definition['%s hours'] = '%s hours';
 $Definition['%s Insightfuls'] = '%s Insightfuls';
+$Definition['%s is'] = '%s is';
```
Comment (tburry) [inline tx-source/site_core.php:54]: This is probably not safe to translate this way. Other language cannot create sentences the same way as English due to masculine/feminine words and verb conjugation.

File: tx-source/site_core.php [hunk 4]
```diff
@@ -83,6 +84,7 @@
 $Definition['%s Points'] = '%s Points';
 $Definition['%s points'] = '%s points';
 $Definition['%s posts'] = '%s posts';
+$Definition['%s prohibited from using the ignore feature.'] = '%s prohibited from using the ignore feature.';
```
Comment (tburry) [inline tx-source/site_core.php:87]: Is this string still there?

**Pull request merged**